dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -601,7 +603,8 @@ def _get_categories_full(self, df_series) -> dict:
         :return: dict of counts for each unique value
         :rtype: dict
         """
-        category_count: dict = df_series.value_counts(dropna=False).to_dict()
+        value_counts = df_series.value_counts(sort=True)
+        category_count: dict = dict(value_counts.iter_rows())
```

Comment:
```suggestion 
        category_count: dict = dict(df_series.value_counts(sort=True).iter_rows())
```
why not 1 line? Does this not run?